apps/explorer/src/app/components/txs/details/liquidity-provision/liquidity-provision-details.spec.tsx

```diff
@@ -0,0 +1,209 @@
+import { MockedProvider } from '@apollo/client/testing';
+import { render } from '@testing-library/react';
+import type { LiquidityOrder } from '@vegaprotocol/types';
+import { PeggedReference } from '@vegaprotocol/types';
+import type { LiquiditySubmission } from '../tx-liquidity-submission';
+import {
+  LiquidityProvisionDetails,
+  sumProportions,
+} from './liquidity-provision-details';
+
+function mockProportion(proportion: number): LiquidityOrder {
+  return {
+    proportion,
+    reference: PeggedReference.PEGGED_REFERENCE_MID,
+    offset: '1',
+  };
+}
+describe('Scale proportions - LP proportions do not always add up to 100%', () => {
+  it('returns 0 if the side is undefined', () => {
+    const side: LiquidityOrder[] = undefined as unknown as LiquidityOrder[];
+    const res = sumProportions(side);
+
+    expect(res).toEqual(0);
+  });
+
+  it('returns 0 if the side is empty', () => {
+    const side: LiquidityOrder[] = [];
+    const res = sumProportions(side);
+
+    expect(res).toEqual(0);
+  });
+
+  it('Sums 1 item correctly (under 100%)', () => {
```

Comment:
(totally non-blocking): I think this has been raised before, and I am perhaps super peaky, but `it('...` test titles get very inconsistent across this file. Sometimes they are titlecase and sometimes they are not. 

I usually go 
```ts
describe('<name-of-component-or-function>'...
   it('should ... 
       // or any kind of verb really that would make the test case readable when we run the whole test suite
       // this way tests could serve as readable acceptance criteria or documentation 

```
Example: 
![image](https://user-images.githubusercontent.com/16125548/213261198-c40a993f-26a9-44b5-a559-b9b7090950a5.png)